profile/profile.go

```diff
@@ -107,6 +108,8 @@ func StartContinuousProfile(profileDir string) {
 			// keeps the size of the log small while still providing lots of
 			// information.
 			StopCPUProfile()
+			runtime.GC()
+			debug.FreeOSMemory()
```

Comment:
yeah I think I agree here, probably not the right choice. I'll pull out the commit.